redis/mutation_utils.cc

```diff
@@ -78,11 +102,63 @@ mutation make_mutation(service::storage_proxy& proxy, const redis_options& optio
     return m;
 }
 
-future<> write_strings(service::storage_proxy& proxy, redis::redis_options& options, bytes&& key, bytes&& data, long ttl, service_permit permit) {
+future<bool> write_strings(service::storage_proxy& proxy, redis::redis_options& options, bytes&& key, bytes&& data, long ttl, service_permit permit, smp_service_group& ssg) {
+    // construct cas_request
     db::timeout_clock::time_point timeout = db::timeout_clock::now() + options.get_write_timeout();
-    auto m = make_mutation(proxy, options, std::move(key), std::move(data), ttl);
+    auto schema = get_schema(proxy, options.get_keyspace_name(), redis::STRINGs);
```

Comment:
Nitpick (maybe out of this patch's scope): Why do we need to search for the schema again and again on every request, instead of saving "strings_schema" once and using it?

Reply:
Leave the issue for now, will handle it on another PR.